app/assets/stylesheets/views/_listings.css.scss

```diff
@@ -229,7 +229,11 @@ $listing-author-avatar-height: 108;
 // Transaction action button
 .book-button .content {
   width: 100%;
-  display: block;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
```

Comment:
This seems to work 👍  
(Personally, I'm pretty annoyed by our SASS `@extend` code, which makes it very hard to find correct code from haml-files. I.e. there is no `.book-button` but `.enabled-book-button`.)